dev/parm/config/gfs/config.base.j2

```diff
@@ -435,6 +435,24 @@ fi
 
 export DO_GSISOILDA="{{ DO_GSISOILDA }}"
 
+# Land IAU for soil
+export DO_LAND_IAU="{{ DO_LAND_IAU }}"
+export LAND_IAU_FHRS="3,6,9"
+export LAND_IAU_DELHRS=6
+export LAND_IAU_INC_FILES='sfc_inc',''
+export LSOIL_INCR=3
+export LAND_IAU_FILTER_INCREMENTS=.false.
+export LAND_IAU_UPD_STC=.true.
+export LAND_IAU_UPD_SLC=.true.
+export LAND_IAU_DO_STCSMC_ADJUSTMENT=.true.
+export LAND_IAU_MIN_T_INCREMENT=0.0001
+#export LAND_IAU_MIN_SLC_INCREMENT=0.000001
```

Comment:
These should be in `config.fcst` as they are used only in the forecast model configuration.